src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+    private double height;
+
+    public IsoscelesTrapezoid(String color,
+                              double firstLeg,
+                              double secondLeg,
+                              double height) {
```

Comment:
When passing parameters to the constructor, it's better to use separate variables for clarity rather than expressions. This is currently not an issue here, but keep it in mind for where you generate these values.